package.json

```diff
@@ -101,6 +101,7 @@
     "@aws-sdk/client-s3": "^3.460.0",
     "@ministryofjustice/frontend": "^1.8.1",
     "@sentry/node": "^7.82.0",
+    "@types/qs": "^6.9.11",
```

Comment:
this ought to go into the dev dependencies section. anything to do with typescript compilation and validation should go there as it doesn't remain in the real running application